site/src/lib/api/blocks/npm.ts

```diff
@@ -135,7 +132,7 @@ export const publishBlockFromNpm = async (
 
   const blockLinkedToPackage = await getDbBlock({ npmPackageName });
 
-  if (isProduction && blockLinkedToPackage) {
+  if (blockLinkedToPackage) {
```

Comment:
I brought back `isProduction` in https://github.com/blockprotocol/blockprotocol/pull/565/commits/7b0c27f8bae1f3a8166a8735de282e53bcf04774 but it [broke](https://github.com/blockprotocol/blockprotocol/actions/runs/3135179464/jobs/5090563570#step:9:48) one test:

```
integration/block-publish-npm.test.ts:90:1 › user should not be able to publish an already-used npm-package 
```

I reverted the change for CI to pass (https://github.com/blockprotocol/blockprotocol/pull/565/commits/26ea28a0698e7a2e9470487d5e896824210277d1). TBH keeping the behaviour consistent across envs makes sense to me. If we want preview deployments to act differently, we should probably introduce a feature flag. This will allow us to test needed behaviors locally and avoid surprises in prod.

Reply:
Okay, let's leave it like this for now (as you have it), we can tweak it later as needed for dev purposes.